screens/TabOneScreen.tsx

```diff
@@ -21,12 +22,41 @@ const styles = StyleSheet.create({
   }
 })
 
-export default function TabOneScreen ({ navigation }: RootTabScreenProps<'TabOne'>) {
+export default function TabOneScreen () {
+  const [client, setClient] = useState<WebDAVClient>()
+
+  async function getSettings () {
+    try {
+      const serverUrl = await AsyncStorage.getItem('serverUrl')
+      const username = await AsyncStorage.getItem('username')
+      const password = await AsyncStorage.getItem('password')
+      if (serverUrl !== null && username !== null && password !== null) {
```

Comment:
suggestion: don't create a webdav client if empty string

```suggestion
      if (!serverUrl && !username && !password) {
```